### PR TITLE
Automatically tag the main branch

### PR DESCRIPTION
**Pull request by Aurora12**
Creates a tag on remote when a commit is pushed to the main branch by calling a script in the root of the repo. A new tag is only created if no version tag matches the version stored in VERSION.txt.

### CODE REVIEW

File: tag_main_branch.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+MAIN_BRANCH=master
+
+echo INDENTED FOR USE WITH A GITHUB ACTION 
+echo TRIGGERED BY A PR MERGE INTO THE MAIN \($MAIN_BRANCH\) BRANCH!
+
+BRANCH=$(git symbolic-ref -q --short HEAD)
+if [ "$BRANCH" != "$MAIN_BRANCH" ]; then	
+	echo Error: not the main branch of the repo!
```
Comment (ygworldr) [inline tag_main_branch.sh:10]: This is not an error, is it?

In any case, this should be redundant if the action only ever runs on master/main.

**Review by ygworldr — COMMENTED** [review]
You could run [beautysh](https://github.com/lovesegfault/beautysh) and [ShellCheck](https://github.com/koalaman/shellcheck) if you wished to.

File: tag_main_branch.sh (hunk 1 above)
Comment (Aurora12) [reply tag_main_branch.sh]: If somebody runs it manually, they will see it was a wrong thing to do. 
I made it better: now the main branch is a parameter. This way we can use this same script in all of the repos!

**Comment by Aurora12** [comment]
> You could run [beautysh](https://github.com/lovesegfault/beautysh) and [ShellCheck](https://github.com/koalaman/shellcheck) if you wished to.

Used shellcheck

**Review by ygworldr — APPROVED** [review]
LGTM

**Pull request merged**